<ui version="4.0" >
 <class>ConfigPlugin</class>
 <widget class="QDialog" name="ConfigPlugin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configurar Efectos</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" >
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Server Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Direccion IP:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="le_ip" />
        </item>
        <item row="1" column="0" >
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="QCheckBox" name="cb_allips" >
          <property name="text" >
           <string>Escuchar en todas las interfaces</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Puerto:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="sb_port" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>65534</number>
          </property>
          <property name="value" >
           <number>6050</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Solo el administrador puede seleccionar entre 1 y 1024</string>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Actualizar Imagen cada:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="sb_seconds" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>3600</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>segundos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="label_11" >
     <property name="text" >
      <string>Este plugin de ejemplo, puede mejorarse agregando whitelists de acceso, keep-alive, data save, etc.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="tl_currentip" >
     <property name="text" >
      <string>---</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer name="horizontalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>206</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="pushButton" >
     <property name="text" >
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="pushButton_2" >
     <property name="text" >
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_ip</tabstop>
  <tabstop>cb_allips</tabstop>
  <tabstop>sb_port</tabstop>
  <tabstop>sb_seconds</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPlugin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>260</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPlugin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>362</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel" >
     <x>370</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
